calendar_event_link_project_task/views/project_task.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="project_task_form_view">
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_form2"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='button_box']" position="inside">
+<button class="oe_stat_button" type="object" name="action_show_events" icon="fa-calendar">
+    <div class="o_stat_info">
+        <field name="event_count" class="o_stat_value"/>
+        <span class="o_stat_text" attrs="{'invisible': [('event_count', '&lt;', 2)]}"> Meetings</span>
```

Comment:
The indent is still wrong.

Nit:
One span has a space (` Meetings`) the other not (`Meeting`).

Reply:
```suggestion
        <span class="o_stat_text" attrs="{'invisible': [('event_count', '&lt;', 2)]}">Meetings</span>
```